Certificates/cert-gen.py

```diff
@@ -0,0 +1,103 @@
+# Requires inkscape in path and PDF module
+import argparse
+import csv
+import numpy as np
+import os
+import pathlib
+from PDF import PdfFileReader, PdfFileWriter
+from subprocess import run
+
+def argument_parser():
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        'template_svg',
+        help="Input svg template with text matching 'NAME OF STUDENT IN CAPS' and 'NAME OF SCHOOL IN CAPS' for "
+             "replacement with the students' names and schools'/teams' names, respectively, "
+             "on the produced certificates.",
+        type=pathlib.Path,
+    )
+    parser.add_argument(
+        '-i', '--input_data',
+        help="Input csv with columns of 'First Name', 'Last Name', and 'School'."
+             "In absence of this argument, the script's own test data will be used.",
+        nargs=1,
+        type=pathlib.Path,
+    )
+    parser.add_argument(
+        '-o','--output_path',
+        help="Path in which the output will be deposited. Defaults to current working directory.",
+        type=pathlib.Path,
+        nargs=1,
+        default=os.getcwd(),
+    )
+    return parser
+
+def main():
+    options = argument_parser().parse_args()
+    
+    outpath = pathlib.Path(options.output_path, "Certificates.pdf")
+    tempsvgpath = pathlib.Path(options.output_path, ".temp-cert.svg")
```

Comment:
You might enjoy the notation `pathlib_path / "my.file"` here -- I find the overriding of the `/` operator one of the most pythonic aspects of `pathlib`.
``` python
>>> pathlib.Path('/foo') / 'bar' / 'spam.quox'
PosixPath('/foo/bar/spam.quox')
```

Reply:
Indeed, I'd rather forgotten about that :) I've fallen into a bad habit of not bothering with using pathlib at all of late, since virtually all my development has been personal projects